ramls/request.json

```diff
@@ -433,6 +433,17 @@
       "description": "Metadata about creation and changes to requests, provided by the server (client should not provide)",
       "type": "object",
       "$ref": "raml-util/schemas/metadata.schema"
+    },
+    "requestProcessingParameters": {
+      "type": "object",
+      "description": "Additional parameters used for request processing and discarded afterwards. Not part of request record.",
```

Comment:
Thank you for making it explicit that these properties are used for directing the process and not part of the record representation.